minecraft/common/src/main/java/dev/kosmx/playerAnim/mixin/PlayerModelAccessor.java

```diff
@@ -0,0 +1,12 @@
+package dev.kosmx.playerAnim.mixin;
+
+import net.minecraft.client.model.PlayerModel;
+import net.minecraft.client.model.geom.ModelPart;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+@Mixin(PlayerModel.class)
+public interface PlayerModelAccessor {
+    @Accessor
+    ModelPart getCloak();
+}
```

Comment:
Do we want to replace the accessot with an AW?

```
accessible field net/minecraft/client/model/PlayerModel cloak Lnet/minecraft/client/model/geom/ModelPart;
```

Reply:
Last time I tried using AWs for a library made using Architectury Loom it caused a crash when depended on by another project
The current solutions is fine imo